core-services/supplemental-ci-images/boskos.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: boskos
+  namespace: ci
+spec:
+  tags:
+  - name: latest
+    from:
+      kind: DockerImage
+      name: gcr.io/k8s-staging-boskos/boskos:latest
```

Comment:
Can this suffer from the skew against the production boskos which is version pinned?l

Reply:
Also, seeing `pj-rehearse` failing with:

```
time="2020-09-16T11:33:47Z" level=error msg="Failed to set up dependencies" error="failed to ensure imagestreamtags in cluster build01: failed waiting for imagestreamtag ci/boskos:latest to appear: timed out waiting for the condition" org=openshift repo=release 
```

makes me thinking this should go it separately so that we get the images building job rehearsal, WDYT?